root/static/scripts/common/constants/locales.json

```diff
@@ -141,6 +191,7 @@
   "el":"Greek",
   "el_CY":"Greek Cyprus",
   "el_GR":"Greek Greece",
+  "el_polyton":"Greek",
```

Comment:
This seems confusing.

Reply:
Found this about Polytonic Greek: https://en.wikipedia.org/wiki/Greek_diacritics#Computer_encoding

I’m not sure how relevant this locale is since MusicBrainz uses Unicode.

Reporting the ambiguous name issue upstream.

I suggest to just skip it for now:
```suggestion
```